packages/terra-application-header-layout/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "terra-application-header-layout",
+  "main": "lib/ApplicationHeaderLayout.js",
+  "version": "0.0.0",
+  "description": "This component renders an application header template. To be used with a layout.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-application-header-layout",
+    "Application",
+    "Header",
+    "Layout",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-router-dom": "^4.0.0",
```

Comment:
You probably don't need these react-router-dom dependencies here or in the menu layout